source/renderer/app/components/settings/categories/WalletsSettings.scss

```diff
@@ -79,3 +77,19 @@
     margin: 15px 0;
   }
 }
+
+:global
+  .WalletsSettings_component
+  .OptionsOverrides_options
+  :not(.SimpleOptions_hasSearch)
+  ul {
+  border-radius: 0 0 4px 4px;
+}
+
+:global {
+  .WalletSettings_component {
+    .SimpleOptions_option em {
+      background-color: var(--rp-option-search-highlight-background-color) !important;
+    }
+  }
+}
```

Comment:
Please nest the `:global` rule within `.component` selector (then you can remove the hardcoded reference to `.WalletSettings_component` like this:

```scss
.component {
  :global {
      .SimpleOptions_option em {
        background-color: var(--rp-option-search-highlight-background-color) !important;
      }
  }
}
```

same for all other places of course 🙏

Reply:
Fixed

Reply:
@aleksandardjordjeviciohk at the bottom of this css file is another `:global` that should be refactored the same was 🙏

Reply:
Fixed

Reply:
@aleksandardjordjeviciohk this still hasn't been fixed. Please **remove** global declaration from here!

Reply:
https://github.com/input-output-hk/daedalus/blob/feature/ddw-645-update-react-polymorph-and-implement-the-search-style-variables/source/renderer/app/components/settings/categories/WalletsSettings.scss#L80-L91

Reply:
@aleksandardjordjeviciohk please fix these 🙏

Reply:
Fixed @nikolaglumac

Reply:
@nikolaglumac @DominikGuzei fixed